src/main/res/values/strings.xml

```diff
@@ -63,8 +63,10 @@
     <string name="remote_service_started">SPV service started</string>
     <string name="remote_service_stopped">SPV service stopped</string>
     <string name="local_service_label">SPV Service</string>
-    <string name="notification_peers_connected_msg">%d peers connected.</string>
-    <string name="notification_no_peers_connected_msg">No peers connected.</string>
+    <plurals name="notification_peers_connected_msg">
+        <item quantity="zero">No peers connected</item>
```

Comment:
Didn't you find that "zero" doesn't work? How about adding the "one" case, as it works and matters (a bit).

Reply:
This would work for another languages, but would add one too.